html-css/week1/Ali-Nematollahi/Order-Desktop-pure-css/index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>order-desktop</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="layout-container">
+
+      <!-- Order Summary Part -->
+
+      <section class="gradient">
+        <h2 class="order-summary">Order Summary</h2>
+        <p class="order-summary-part1">
+          <span class="">Total</span>
+          <span class="left-view">$333 / month</span>
+        </p>
+        <p class="order-summary-part2">
+          <span class="">Total</span>
+          <span class="left-view">$100</span>
+        </p>
+      </section>
+
+      <!-- Payment Method Part -->
+
+      <section class="payment container-fluid">
+        <h2 class="section-title">Payment Method</h2>
+        <div class="payment-container">
+          <div class="payment-option">
```

Comment:
This should be `<button>` as theoretically, it should be clickable.
```suggestion
          <button class="payment-option" type="button">
```